README.md

```diff
@@ -53,6 +53,14 @@ Usage: relation-graph [options]
         Set log level to INFO
 ```
 
+## Installation
+### MacOS
+1. Download `relation-graph-cli-*.tgz` [latest release](https://github.com/balhoff/relation-graph/releases/latest).
+2. Unzip `relation-graph-cli-*.tgz`.
+3. From within the unzipped folder, run:
```

Comment:
I don't know if there's a more sophisticated way to do this, but this is what I did.